ATSketchKit/ATSketchView.swift

```diff
@@ -22,19 +22,19 @@
 
 import UIKit
 
+public enum Tools {
```

Comment:
Please leave the tools inside the view. It makes more sense as they are only used inside the ATSketchView.
And yes, I know this means changing everything else :stuck_out_tongue_closed_eyes: